staticd/static_nb_config.c

```diff
@@ -136,11 +136,11 @@ static bool static_nexthop_create(struct nb_cb_create_args *args)
 	case NB_EV_VALIDATE:
 		ifname = yang_dnode_get_string(args->dnode, "./interface");
 		if (ifname != NULL) {
-			if (strcasecmp(ifname, "Null0") == 0
```

Comment:
Can I get a better explanation of what is going on here?  Null0 is a perfectly valid value to use.

Reply:
as an aside, I'm pretty paranoid about anything that touches Null0.  We had too many issues with it at the start of this whole thing.  Kept on getting it wrong and it seems like it's in a good place now and I'm hesitant to touch anything related to Null0

Reply:
This condition is never valid because if we set Null0 as a nexthop, it's `ifname == NULL`.